src/js/components/FormField/stories/TextInput.js

```diff
@@ -14,7 +14,8 @@ class FormFieldTextInput extends Component {
     const {
       target: { value },
     } = event;
-    const exp = new RegExp(value, 'i');
+    const escapedText = value.replace(/[-\\^$*+?.()|[\]{}]/g, '\\$&');
```

Comment:
Can we add a quick comment of what does this regexp do, and what problem does it solve? It might be confusing or overwhelming for users that don't have the context.

Reply:
Yes, done!